### DISCUSSION TITLE
Default Currency

### DISCUSSION THREAD

**Discussion by maulwurf87** [cmintey/wishlist#184] [Q&A] [answered]
Hi,
is there a list of supported default currencies? I wanted to configure Euro, but I am not able to get it displayed in wishlist.
My docker-compose.yaml has this ENV:
      DEFAULT_CURRENCY: EUR
Is that correct?

**Comment by cmintey** [comment, marked as answer]
Hi there. Yeah that's the correct env variable and currency code. Once you set it, you need to restart the container. The default currency only applies to new items and will be used if the currency couldn't be extracted from the item data. 

You can change the currency on individual items. The currency field is editable.

**Comment by maulwurf87** [comment]
Thanks for the fast reply. I messed up the recreating container part. Great how things just work if you do them right. :-)
Love the app so far. I love the autodiscover of the relevant data. Really makes it easy to use.
